share/spice/forecast/forecast.js

```diff
@@ -158,10 +148,8 @@
         dailyObj[i] = days[i];
         day = days[i];
 
-        tmp_date = new Date();
-        tmp_date.setDate(date_i+i);
-        dailyObj[i].date = tmp_date.toDateString().substr(4,6);
-        dailyObj[i].day = i == 0 ? 'Today' : day_strs[(today_i+i)%7];
+        tmp_date = moment(today).add(i, 'days');
+        dailyObj[i].date = tmp_date.format("MMM D");        
```

Comment:
There is a subtle formatting difference here. Previously single digit dates would be prefixed with zero (eg: `01 Feb`), whereas now they're displayed as `1 Feb`